android/ndk_common.sh

```diff
@@ -7,14 +7,22 @@ export NDK_ARMV6_ROOT=$BUILD_DIR/android-ndk-r${NDK_ARMV6_VERSION}
 
 createNDKFolder()
 {
+    ndk_filename=android-ndk-r${NDK_VERSION}-linux-x86_64.zip
     rm -rf "$BUILD_DIR/android-ndk-r${NDK_VERSION}"
+    echo Downloading $ndk_filename
     wget -c --no-verbose -O /tmp/ndk_${NDK_VERSION}.zip https://dl.google.com/android/repository/android-ndk-r${NDK_VERSION}-linux-x86_64.zip
```

Comment:
You can reuse new variable here.
E.g.

wget -c --no-verbose -O /tmp/${ndk_filename} https://dl.google.com/android/repository/${ndk_filename}